scripts/lib/CIME/XML/batch.py

```diff
@@ -91,7 +91,7 @@ def get_value(self, name, attribute=None, resolved=True, subgroup=None):
 
         node = self.get_optional_batch_node(name)
         if node is not None:
-            value = node.text
+            value = node.text()
```

Comment:
Interesting - is this change just for good form?  It seems to function correctly as is.

Reply:
I had to do this since nodes are ConstantElements in this PR.